po/it.po

```diff
@@ -0,0 +1,240 @@
+# Esrille Inc. <info@esrille.com>, 2020-2023.
+# Albno Battistella <albanobattistella@gmail.com>,2024.
+# 
+msgid ""
+msgstr ""
+"Project-Id-Version: furiganapad 0.7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-05-17 22:09+0900\n"
+"PO-Revision-Date: 2024-05-21 14:38+0100\n"
+"Last-Translator: Albano Battistella <albanobattistella@gmail.com>\n"
+"Language-Team: Italian <albanobattistella@gmail.com>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/com.esrille.furiganapad.desktop.in:4
+#: data/com.esrille.furiganapad.metainfo.xml.in:6 src/window.py:45
+msgid "FuriganaPad"
+msgstr "FuriganaPad"
+
+#: data/com.esrille.furiganapad.desktop.in:5
+#: data/com.esrille.furiganapad.metainfo.xml.in:7
+msgid "Write Japanese text with Ruby annotations easily"
+msgstr "Scrivi facilmente testo giapponese con annotazioni Ruby"
+
+#: data/com.esrille.furiganapad.metainfo.xml.in:12
+msgid ""
+"With Hiragana IME for IBus, FuriganaPad automatically puts Ruby characters "
+"above Kanji as you type them."
+msgstr ""
+"Con Hiragana IME per IBus, FuriganaPad inserisce automaticamente i caratteri Ruby "
+"sopra i Kanji mentre li digiti."
+
+#: data/com.esrille.furiganapad.metainfo.xml.in:24
+msgid "Screenshot"
+msgstr "Screenshot"
+
+#: data/com.esrille.furiganapad.metainfo.xml.in:28
+msgid "Screen-capture"
+msgstr "Cattura dello schermo"
+
+#: src/application.py:49
+msgid "Initial window x position"
+msgstr "Posizione iniziale della finestra x"
+
+#: src/application.py:49
+msgid "x"
+msgstr "x"
+
+#: src/application.py:51
+msgid "Initial window y position"
+msgstr "Posizione iniziale della finestra y"
+
+#: src/application.py:51
+msgid "y"
+msgstr "y"
+
+#: src/application.py:53
+msgid "Initial window width"
+msgstr "Larghezza iniziale della finestra"
+
+#: src/application.py:53
+msgid "w"
+msgstr "w"
+
+#: src/application.py:55
+msgid "Initial window height"
+msgstr "Altezza iniziale della finestra"
+
+#: src/application.py:55
+msgid "h"
+msgstr "h"
+
+#: src/application.py:57
+msgid "Print version information"
+msgstr "Stampa le informazioni sulla versione"
+
+#: src/window.py:99 src/window.py:529 src/window.py:531
+msgid "Ruby"
+msgstr "Ruby"
+
+#: src/window.py:207
+msgid "Text files"
+msgstr "File di testo"
+
+#: src/window.py:212
+msgid "Python files"
+msgstr "File Python"
+
+#: src/window.py:217
+msgid "Any files"
+msgstr "Qualsiasi file"
+
+#: src/window.py:235
+msgid "Save changes to this document?"
+msgstr "Vuoi salvare le modifiche a questo documento?"
+
+#: src/window.py:236
+msgid "If you don't, changes will be lost."
+msgstr "In caso contrario, le modifiche andranno perse"
+
+#: src/window.py:237
+msgid "Close _Without Saving"
+msgstr "Chiudi _Senza Salvare"
+
+#: src/window.py:270
+msgid "Font"
+msgstr "Font"
+
+#: src/window.py:271
+msgid "The quick brown fox jumps over the lazy dog."
+msgstr "La veloce volpe marrone salta sopra il cane pigro."
```

Comment:
We don’t need to use a direct translation here. What’s the popular pangram in Italy these days?
See https://it.wikipedia.org/wiki/The_quick_brown_fox_jumps_over_the_lazy_dog

Reply:
> We don’t need to use a direct translation here. What’s the popular pangram in Italy these days? See https://it.wikipedia.org/wiki/The_quick_brown_fox_jumps_over_the_lazy_dog

HI,
there is no specific one but one as written is:"Pranzo d'acqua fa volti sghembi" or "Ma la volpe col suo balzo ha raggiunto il quieto Fido"

Reply:
Will you please pick one that is often used in the font selection dialog in Italy?